src/frontend-samples/fire-decoration-sample/README.md

```diff
@@ -14,7 +14,7 @@ The purpose of this sample is to demonstrate the following:
 
 ## Description
 
-This sample provides an example of a particle effect in world space (as apposed to view space).  The sample provides features to place and configure fire particle emitters.  Each emitter is initially configured based on preset parameters defined in [FireDecorationApp.ts](./FireDecorationApp.tsx).  Select which preset is used with the drop down menu.  Once you have deselected an emitter, there is no way to return to configuring it.
+This sample provides an example of a particle effect in world space (as opposed to view space).  The sample provides features to place and configure fire particle emitters.  Each emitter is initially configured based on preset parameters defined in [FireDecorationApp.ts](./FireDecorationApp.tsx).  Select which preset is used with the drop down menu.  Once you have deselected an emitter, there is no way to return to configuring it.
```

Comment:
This sample does not demonstrate a way to configure an emitter after it has been deselected.